lib/managers/files.js

```diff
@@ -1359,10 +1359,19 @@ Files.prototype.representation = {
  * @param {string} fileID - Box ID of the file being requested
  * @param {client.files.representation} representationType - The x-rep-hints value the application should create a
  *    representation for. This value can either come from FileRepresentationType enum or manually created
+ * @param {Object} [options] - Additional options. Can be left empty
+ * @param {Boolean} [options.generateRepresentations] - Set to true to return representation info where all states resolve to success. Default is false
```

Comment:
The type for this should be `{bool}` instead of `{Boolean}` — we expect it to be the primitive, not the object type.  You can also indicate the default value like this: `[options.generateRepresentations=false]`